BUILDING.md

```diff
@@ -11,7 +11,7 @@ CMake provides a portable cross-platform build systems with many useful features
 VTR requires a C++-14 compliant compiler.
 The following compilers are tested with VTR:
 
-* GCC/G++: 5, 6, 7, 8, 9, 10, 11
+* GCC/G++: 7, 8, 9, 10, 11
 * Clang/Clang++: 6, 7, 10
```

Comment:
Do you want a comment about which versions of compilers we plan to support (based on my comments)?

Reply:
Something like this?

Reply:
LGTM!